docs/tutorials/unsupervised_learning/gan.md

```diff
@@ -86,25 +86,25 @@ Since the DCGAN that we're creating takes in a 64x64 image as the input, we'll u
 import cv2
 X = np.asarray([cv2.resize(x, (64,64)) for x in X])
 ```
-Each pixel in our 64x64 image is represented by a number between 0-255, that represents the intensity of the pixel. However, we want to input numbers between -1 and 1 into our DCGAN, as suggested by the research paper. To rescale our pixels to be in the range of -1 to 1, we'll divide each pixel by (255/2). This put our images on a scale of 0-2. We can then subtract by 1, to get them in the range of -1 to 1.
+Each pixel in the 64x64 image is represented by a number between 0-255, that represents the intensity of the pixel. However, we want to input numbers between -1 and 1 into the DCGAN, as suggested by the research paper. To rescale the pixels to be in the range of -1 to 1, we'll divide each pixel by (255/2). This put the images on a scale of 0-2. We can then subtract by 1, to get them in the range of -1 to 1.
```

Comment:
Have we linked the research paper anywhere? If not, this would be a good place to do that.

This seems a little verbose. See if this sounds better:
"as suggested by the research paper. To rescale the pixels to be in the range of -1 to 1, we'll divide each pixel by (255/2). This put the images on a scale of 0-2. We can then subtract by 1, to get them in the range of -1 to 1." => 
"as suggested by the research paper. To rescale the pixel values, we'll divide it by (255/2). This changes the scale to 0-2. We then subtract by 1 to get them in the range of -1 to 1."